pocs/camera/canon_gphoto2.py

```diff
@@ -139,7 +139,11 @@ def take_observation(self, observation, headers=None, filename=None, **kwargs):
             'start_time': start_time,
         }
         metadata.update(headers)
+
+        # This can be different than for the headers because passed as param
         exp_time = kwargs.get('exp_time', observation.exp_time.value)
```

Comment:
I'm curious why exp_time=None is not explicitly in the parameter list of the function decl, rather than using kwargs as done here.

Reply:
It probably could be but the method is supposed to just take an `observation`, which defines the exp_time in most scenarios. This could be changed but should happen after #363, which consolidates a lot of the camera methods so we only have to change it in one place.